user/apps/l4test/mem.c

```diff
@@ -0,0 +1,39 @@
+/* Copyright (c) 2002, 2003, 2007, 2010 Karlsruhe University.
```

Comment:
You should replace the copyright notice with F9 Microkernel project since it is ARM Cortex-M specific.